src/components/Header.js

```diff
@@ -59,7 +59,7 @@ function Header() {
 
   return (
     <header className="header">
-      <Link to={globalState.token !== null ? '/home' : '/'}>
+      <Link to={globalState.auth_status === true ? '/home' : '/'}>
```

Comment:
It's not necessary to include `=== true`

Reply:
Just refactored the naming for the authorization method, I think now it will be easier to maintain the code